installation/routines/customize_options.sh

```diff
@@ -305,25 +305,37 @@ Do you want to install Node? [Y/n]"
       read -r response
       case "$response" in
         [nN][oO]|[nN])
-          ENABLE_INSTALL_NODE=false
-          ENABLE_WEBAPP_PROD_DOWNLOAD=true
-          ;;
+            ENABLE_INSTALL_NODE=false
+            ENABLE_WEBAPP_PROD_DOWNLOAD=true
+            ;;
         *)
-          # This message will be displayed at the end of the installation process
-          local tmp_fin_message="ATTENTION: You need to build the web app locally with
-           $ cd ~/RPi-Jukebox-RFID/src/webapp && ./run_rebuild.sh -u
-           This must be done after reboot, due to memory restrictions.
-           Read the documentation regarding local Web App builds!"
-          FIN_MESSAGE="${FIN_MESSAGE:+$FIN_MESSAGE\n}${tmp_fin_message}"
+            print_c "
+Building the webapp locally takes some time.
+This can be done now or manually after the installation.
+
+Do you want to build the webapp now? [Y/n]"
```

Comment:
Should we even ask this question? I don't really see a benefit of doing it after the installation. In case we build the webapp locally, we should inform the user about the constrained resources but go ahead and build it directly.

So instead, I would ask in L304 (which is not part of this PR): 'Do you want to build the webapp?'

Reply:
I reverted this and made some more changes to simplify this case.